assets/content/cookbook/Abstract types/working-around-iterator-limitations-as-generic-type-parameter.md

```diff
@@ -0,0 +1,37 @@
+[tags]: / "abstract-type,type-params,iterator"
+
+# Working Around Iterators as a Generic Type Parameter
```

Comment:
I also think the article belongs to "iterators", not "abstract types", not sure though